ddl-introduction.md

```diff
@@ -0,0 +1,254 @@
+---
+title: DDL 介绍
```

Comment:
@wjhuang2016 Please add this file to the TOC.md. Otherwise, this doc will not be dispalyed in the navigation of the documentation website.